### PR TITLE
4.4.12 typo fix

### PR DESCRIPTION
**Pull request by escopecz**
There was a typo in the 4.4.12 release: https://github.com/mautic/recommended-project/pull/68

Issue: https://github.com/mautic/mautic/issues/13647

I'll try to re-tag once this PR is merged and hope that it will fix the release.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -27,43 +27,43 @@
...
-    "mautic/theme-oxygen": "4.4.x-dev",
-    "mautic/theme-paprika": "4.4.x-dev",
-    "mautic/theme-skyline": "4.4.x-dev",
-    "mautic/theme-sparse": "4.4.x-dev",
-    "mautic/theme-sunday": "4.4.x-dev",
-    "mautic/theme-vibrant": "4.4.x-dev",
-    "mautic/theme-trulypersonal": "4.4.x-dev"
+    "mautic/core-composer-scaffold": "4.4.12",
+    "mautic/core-project-message": "4.4.12",
+    "mautic/core-lib": "4.4.4.12",
```
Comment (patrykgruszka) [inline composer.json]: Shouldn't it be `4.4.12` instead of `4.4.4.12`?

Comment (escopecz) [reply composer.json]: Did I do it again? Funny ho a find-replace go wrong. Or sad at this point...

Comment (escopecz) [reply composer.json]: Fixed in the second commit.

**Review by patrykgruszka — APPROVED** [review]
Looks good now :)

**Pull request merged**